### PR TITLE
#728 Update documentation related to existing filters

### PR DESCRIPTION
**Pull request by nmadruga**

### CODE REVIEW

File: documentation/internal/Filters-for-Applications.md [hunk 1]
```diff
@@ -8,10 +8,12 @@ The list of applications to display for the current user is based on:
 - the application type (aka template code)
 - other optional filters selected
 
-The user has access to links to see each type of application list from the top menu or in their dashboard - the link will set basic filters in the main URL route of the list: `/applications`. For example the list to see the Application of type "Drug Registration" would take the user to `/applications?type=user-registration&user-role=applicant`.
+The user has access to links to see each type of application list from the top menu or from their dashboard - the link will set basic filters in the main URL route of the list: `/applications`.
+For example to see the list of Applications for "Drug Registration" applyed by you has this link: `/applications?type=user-registration&user-role=applicant`.
```
Comment (CarlosNZ) [inline documentation/internal/Filters-for-Applications.md:12]:
```suggestion
For example to see the list of Applications for "Drug Registration" applied by you has this link: `/applications?type=user-registration&user-role=applicant`.
```
sp

File: documentation/internal/Filters-for-Applications.md [hunk 2]
```diff
@@ -57,59 +62,56 @@ The `user-role` is used to determine how the columns and rows should be displaye
...
-Options:
-
-- Set of `status` options
+- `draft` or `DRAFT`
+- `withdrawn` or `WITHDRAWN` _not implemented_
+- `submitted` or `SUBMITTED`
+- `changes required` or `CHANGES_REQUIRED`
+- `re submitted` or `RE_SUBMITTED` _not implemented_
+- `completed`or `COMPLETED`
+- `expired` or `EXPIRED` _not implemented_
```
Comment (CarlosNZ) [inline documentation/internal/Filters-for-Applications.md:106]: Update these to new values (Withdrawn and Expired moved to Outcomes)

File: documentation/internal/Filters-for-Applications.md [hunk 3]
```diff
@@ -134,83 +136,108 @@ Options:
...
 <a name="applicant"></a>
 
 #### applicant
 
-Example: `applicant="John"`  
-Considered in implementation: **No**  
-Can have combined values: **Yes** e.g: `applicant="John","Nicole"`  
+Included: **Yes**
+Can have combined values: **Yes**
+Exmples: `applicant=Carl` or `applicant=Carl Smith`
```
Comment (CarlosNZ) [inline documentation/internal/Filters-for-Applications.md:225]: Sp: "Examples"
-- also, are spaces allowed in this string?

**Review by CarlosNZ — APPROVED** [review]
Great, thanks. Just a couple of tiny typos, you can merge as soon as fixed.

File: documentation/internal/Filters-for-Applications.md (hunk 2 above)
Comment (nmadruga) [reply documentation/internal/Filters-for-Applications.md]: Good! Updating now

File: documentation/internal/Filters-for-Applications.md (hunk 3 above)
Comment (nmadruga) [reply documentation/internal/Filters-for-Applications.md]: Thank you. Fixed now and replaced TODO as well.

Merging...

**Pull request merged**